nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -387,7 +458,13 @@ def permuted_ols(
        linear model. Neuroimage.
 
     .. [3] Freedman, D. & Lane, D. (1983). A nonstochastic interpretation of
-       reported significance levels. J. Bus. Econ. Stats., 1(4), 292-298
+       reported significance levels. J. Bus. Econ. Stats., 1(4), 292-298.
```

Comment:
Might be a good opportunity to switch to bibtex references for `permuted_ols`, WDYT?

Reply:
Great idea. I've added this in fcf1d2f.

When I propagate the TFCE enhancement to `non_parametric_inference`, I'll do the same there.